From 37b553c463c4c4c85bf3a40a97c7e9c830d460a8 Mon Sep 17 00:00:00 2001
From: Charles Johnson <charles.f.johnson@intel.com>
Date: Mon, 26 Mar 2012 08:00:53 -0700
Subject: Add GRALLOC_MODULE_PERFORM_* to enable Gen Graphics

Add GRALLOC_MODULE_PERFORM_GET_DRM_FD, GRALLOC_MODULE_PERFORM_ENTER_VT
and GRALLOC_MODULE_PERFORM_LEAVE_VT enums to enable Gen Graphics.
This does not changes any of the existing enums used by the IMG graphics
driver.

	modified:   gralloc.h

Change-Id: I86d641e50207a590abdd5684f391fddb2ab26d49
---
 include/hardware/gralloc.h |    8 ++++++++
 1 files changed, 8 insertions(+), 0 deletions(-)

diff --git a/include/hardware/gralloc.h b/include/hardware/gralloc.h
index 9c62242..fad431d 100644
--- a/include/hardware/gralloc.h
+++ b/include/hardware/gralloc.h
@@ -242,6 +242,14 @@ typedef struct gralloc_module_t {
     void* reserved_proc[6];
 } gralloc_module_t;
 
+enum {
+    GRALLOC_MODULE_PERFORM_GET_DRM_MAGIC  = 0x10000000,
+    GRALLOC_MODULE_PERFORM_AUTH_DRM_MAGIC = 0x10000001,
+    GRALLOC_MODULE_PERFORM_GET_DRM_FD     = 0x10000002,
+    GRALLOC_MODULE_PERFORM_ENTER_VT       = 0x10000003,
+    GRALLOC_MODULE_PERFORM_LEAVE_VT       = 0x10000004,
+};
+
 /*****************************************************************************/
 
 /**
-- 
1.7.4.1

